opensrp-app/src/main/java/org/smartregister/sync/helper/LocationServiceHelper.java

```diff
@@ -129,8 +132,8 @@ private String fetchLocationsOrStructures(boolean isJurisdiction, Long serverVer
         return resp.payload().toString();
     }
 
-    private String getMaxServerVersion(List<Location> locations, long serverVersionParam) {
-        long currentServerVersion = serverVersionParam;
+    private String getMaxServerVersion(List<Location> locations) {
+        long currentServerVersion = 0;
```

Comment:
Maybe rename this to `maxServerVersion`?

Reply:
done